windowbuilder.aggrcon

```diff
@@ -34,12 +34,6 @@
     <features name="org.eclipse.wb.swing.doc.user.feature.feature.group">
       <categories href="simrel.aggr#//@customCategories[identifier='General%20Purpose%20Tools']"/>
     </features>
-    <features name="org.eclipse.wb.xwt.feature.feature.group" enabled="false">
```

Comment:
@ptziegler or @merks can you have a look at this PR. I think it introduced errors in the aggregator, perhaps due to missing the change to the simrel.aggr file that went along with removing the features.

![image](https://github.com/eclipse-simrel/simrel.build/assets/679236/ceee4a93-3dc6-4c4e-ad79-fcef4def4dcf)